frontend/src/components/segment-bar.tsx

```diff
@@ -18,18 +21,29 @@ const segmentStyle: CSSObject = {
     display: 'flex',
     alignItems: 'center',
     flexDirection: 'column',
+    '&:first-of-type': {
+        '.title': {
+            paddingLeft: segmentTitlePadding,
+        },
+    },
+    '&:last-of-type': {
+        '.title': {
+            paddingRight: segmentTitlePadding,
+        },
+    },
 }
 
 const SegmentWrapper = styled.div(segmentStyle)
 
-export const Segment: FCWC<SegmentProps> = ({ children, percentage }) => {
-    return <SegmentWrapper css={{ left: `calc(${percentage}% - ${segmentWidth / 2}px)` }}>{children}</SegmentWrapper>
+export const Segment: FCWC<SegmentProps> = ({ children, className, percentage }) => {
+    return <SegmentWrapper className={cx('segment', className)} css={{ left: `calc(${percentage}% - ${segmentWidth / 2}px)` }
+    }> {children}</SegmentWrapper >
 }
 
 export const segmentCircleStyle: CSSObject = {
-    height: '20px',
-    width: '20px',
-    borderRadius: '10px',
+    height: circleDiameter,
```

Comment:
Im assuming this will just interpret `20` as a value in pixels right? rather than the explicit `${circleDiameter}px`

Reply:
yep, this is emotionjs, and it defaults to `px` for bare numbers.